### DISCUSSION TITLE
Should I use proxied record on Cloudflare to point to my coolify deployed service?

### DISCUSSION THREAD

**Discussion by piavgh** [coollabsio/coolify#3966] [Q&A]
Hi,

I have a n8n service deployed with Coolify
On Coolify, I set up the domain of this n8n service to https://n8n.mydomain.xyz

At the moment, I enabled a proxied record on Cloudflare, and it works normally (as I understand, now I have 2 SSL certificates from Traefik and Cloudflare).

But when I tried to set my domain to http://n8n.mydomain.xyz to use only the SSL certificate from Cloudflare, I got 404 not found

What is the best practice when deploying my service with Coolify and using Cloudflare for DNS?
1. Use both `https` on Coolify + proxied record on Cloudflare (2 SSL certificates in this case)
2. Use `http` on Coolify + proxied record on Cloudflare (I tried and got 404)
3. Use `https` on Coolify + DNS only record on Cloudflare (as I remember this also works)

Thanks

**Comment by luk82** [comment]
Try to configure ports in Docker for n8n:

1. Navigate to n8n > Production > n8n > Configuration > Service Stack.
2. Click the "Edit Compose File" button.
3. Add the following under the service configuration:
ports:
  - '5678:5678'
4. Save the changes and restart the n8n instance.
